### ISSUE TITLE
source distribution created via setup.py is broken

### ISSUE THREAD

**Issue by marscher** [choderalab/bhmm#46]
```
marscher@jigsaw:~/sources/bhmm$ pip install dist/bhmm-0.3.0.tar.gz 
DEPRECATION: --download-cache has been deprecated and will be removed in the future. Pip now automatically uses and configures its cache.
Processing ./dist/bhmm-0.3.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/tmp/pip-dFq9UY-build/setup.py", line 163, in <module>
        ext_modules = cythonize(extensions)
      File "/home/marscher/anaconda/lib/python2.7/site-packages/Cython/Build/Dependencies.py", line 754, in cythonize
        aliases=aliases)
      File "/home/marscher/anaconda/lib/python2.7/site-packages/Cython/Build/Dependencies.py", line 649, in create_extension_list
        for file in nonempty(extended_iglob(filepattern), "'%s' doesn't match any files" % filepattern):
      File "/home/marscher/anaconda/lib/python2.7/site-packages/Cython/Build/Dependencies.py", line 103, in nonempty
        raise ValueError(error_msg)
    ValueError: './bhmm/hidden/impl_c/hidden.pyx' doesn't match any files

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-dFq9UY-bui
```

It seems you do not include your cython files in the Manifest.in file

**Comment by marscher** [comment]
fixed in #45

**Comment by marscher** [comment]
done
